src/ray/core_worker/store_provider/plasma_store_provider.h

```diff
@@ -151,6 +158,8 @@ class CoreWorkerPlasmaStoreProvider {
              absl::flat_hash_map<ObjectID, std::shared_ptr<RayObject>> *results,
              bool *got_exception);
 
+  Status GetRelease(const ObjectID &object_id);
```

Comment:
Can you add comment when this call can be blocking? (if it is called on an object that's not writtne iiuc? )